pyanaconda/storage/checker.py

```diff
@@ -411,6 +412,51 @@ def verify_mounted_partitions(storage, constraints, report_error, report_warning
                                "installation. Please unmount it and retry.") % part.path)
 
 
+def verify_lvm_destruction(storage, constraints, report_error, report_warning):
+    """ Verify that destruction of LVM devices is correct.
+
+    This means: If a PV is destroyed, the VG on it must be destroyed too.
+    Additionally, destroying an incomplete VG warns about this.
+
+    :param storage: a storage to check
+    :param constraints: a dictionary of constraints
+    :param report_error: a function for error reporting
+    :param report_warning: a function for warning reporting
+    """
+    # Implementation of the verification uses an inverted approach: Check that all PVs to be
+    # destroyed are contained in a VG that will be destroyed too. If there is a PV to be destroyed
+    # that has a VG that isn't going to be destroyed, that is an error. This is because we ask
+    # blivet to ignore some disks, and these might contain a PV. Crucially, blivet implements this
+    # such that the ignoring hides also all dependent devices - for any disk, it could be a PV
+    # partition, which would then hide also all VGs and LVs that depend on the PV. It does not
+    # matter that the VG needs other PVs too - the hiding wins.
+    log.debug("called verify_lvm_destruction()")
+    destroyed_vg_names = set()
+    vg_names_seen_in_destroyed_pvs = set()
+    for action in storage.devicetree.actions:
+        if isinstance(action, ActionDestroyDevice) and action.device.type == "lvmvg":
+            log.debug("ActionDestroyDevice: %s %s", action.device.type, action.device.name)
+            destroyed_vg_names.add(action.device.name)
+            if not action.device.complete:
+                report_warning("Disk selection will result in removing an incomplete volume "
+                               "group '{}'.".format(action.device.name))
```

Comment:
This part is up for discussion - do we want to warn when we'll destroy an incomplete VG? Apparently we can currently just steamroll over that, but it still feels not exactly right.

Reply:
@vojtechtrefny, what do you think?

Reply:
I don't think this message is correct. I would rather see something like:
```Disk selection does not contain all parts of existing volume groups```

Reply:
I don't think it's necessary to warn about this.

Reply:
Removing then.